code-reviews/recipes/Terraform.md

```diff
@@ -1,21 +1,33 @@
 # Terraform Code Reviews
 
-## Style guide
+## Style Guide
 
-[CSE](../../CSE.md) developers adhere to the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html).
+[CSE](../../CSE.md) developers follow the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html).
 
-We also ensure all used providers are versioned to prevent breaking changes in future. [docs on versioning providers](https://www.terraform.io/docs/configuration/providers.html#provider-versions).
+[CSE](../../CSE.md) projects should check Terraform scripts with automated tools.
 
 ## Linting
 
-### tflint
+### TFLint
 
-> TODO: add information and an example of installation and usage of tflint
+[`TFLint`](https://github.com/terraform-linters/tflint) is a Terraform linter focused on possible errors, best practices, etc. Once TFLint installed in the environment, it can be invoked using the VS Code [`terraform extension`](https://marketplace.visualstudio.com/items?itemName=mauve.terraform)
```

Comment:
Nit: Period missing from the end of the sentence.

Reply:
Addressed.

Reply:
updated.